mixins/lint.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+"""
+Lint the index.yaml file as well as all files ending with .mixin.
+Taken from https://github.com/colcon/colcon-mixin-repository.
+Modified for the AWS mixins repository.
```

Comment:
Please make sure that the licenses are compatible.

Reply:
Pre-approved per [Amazon Opensource Preapproved List](https://w.amazon.com/index.php/Open_Source/Distributions/Process/License_Instructions_for_Distribution)